calcom/cal.com
# fix: cal video link on booking confirmation

## What does this PR do?

This PR fixes the bug that caused the generated cal video link on booking confirmation to follow the format 'https://meetco.daily.co/...' instead of the correct format 'https://app.cal.com/video/...'.

Fixes #12941 

 Loom Video: https://www.loom.com/share/87ad5b585f5a49f995fc61c910a2187a?sid=c842b102-3dfb-4dc5-b0c6-b0f336018e8e

## Type of change


## How should this be tested?

- Make a new booking on an event that requires confirmation and select Cal Video as the location type.
- Confirm the booking.
- Check that the newly generated Cal Video link follows the correct pattern - 'http://localhost:3000/video/...'

## Mandatory Tasks
